zppy/defaults/default.ini

```diff
@@ -107,9 +107,11 @@ input_component = string(default="")
 [ts]
 area_nm = string(default="area")
 cmip_metadata = string(default="inclusions/e3sm_to_cmip/default_metadata.json")
+cmip_plevdata = string(default="inclusions/e3sm_to_cmip/vrt_remap_plev19.nc")
```

Comment:
This code needs to be rebased off the latest `main`. `cmip_metadata` isn't handled under `ts` anymore, but rather under the new `e3sm_to_cmip` task.